packages/contracts-bedrock/src/libraries/Predeploys.sol

```diff
@@ -1,6 +1,75 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+interface ICrossL2Inbox {
```

Comment:
Are these interfaces used anywhere? We don't maintain interfaces for the other predeploys

Reply:
ended up creating an interface file for each 0.8.24 contract but in 0.8.15 and stored them in src/L2